### ISSUE TITLE
[FEATURE REQUEST] Option to disable GFE streaming optimizations

### ISSUE THREAD

**Issue by Forceflow** [moonlight-stream/moonlight-chrome#262]
When launching games through the Chrome extension, GFE automatically applies "optimizations" to lower the games settings. This makes pretty much any game look bad.

I know that it's possible disable these optimizations when requesting a stream connection (moonlight-embedded has a command-line switch for it). Could this be added to moonlight-chrome too?

**Comment by Jorys-Paulin** [comment]
#253 Seems like the same problem

**Comment by Jorys-Paulin** [comment]
@Forceflow Hey, I know it's been a while, but I got some news!

- I found [this](https://github.com/moonlight-stream/moonlight-chrome/blob/master/static/js/index.js#L601) in the code:
`1, // Allow GFE to optimise game settings`

- Changing `1` to `0` (or `true` to `false`) should disable Geforce's optimisations. Adding an option in the GUI would fix the problem for the user without running a custom version with that option disabled.

Anyways, hope it helped!
